examples/purchaseTesterTypescript/package.json

```diff
@@ -12,20 +12,21 @@
   },
   "dependencies": {
     "@react-navigation/bottom-tabs": "^6.5.11",
-    "@react-navigation/native": "^6.1.9",
+    "@react-navigation/native": "^6.0.8",
```

Comment:
this is going down?

Reply:
Hmm interesting... was this modified by the upgrade tool @ajpallares